src/excited_workflow/source_datasets/land_cover.py

```diff
@@ -22,7 +22,41 @@ def _cftime_to_datetime(data: xr.DataArray) -> np.ndarray:
     return np.array([np.datetime64(el) for el in data.to_numpy()])
 
 
-def dims_check(dat: xr.Dataset, target: xr.Dataset, var: str) -> bool:
+def regrid(
+    file: Path,
+    target_grid: xr.Dataset,
+    variables: list[str] | None = None,
+) -> xr.Dataset:
+    """Regrids one file to target dataset.
+
+    Args:
+        file: file to regrid
+        variables: List of variable names which should be downloaded.
+        target_grid: Grid to which the data should be regridded to.
```

Comment:
```suggestion
        target_grid: Dataset to which the data should be regridded to.
        variables: List of variable names which should be regridded.
```
they're the other way around in the function definition.